### DISCUSSION TITLE
How is the bureau consent shared with lender today?

### DISCUSSION THREAD

**Discussion by ankitagarwaladobe** [ONDC-Official/ONDC-FIS-Specifications#32] [General]
Generally the bureau consent is shared by OTP verification on the lender side. How does it work in this case?
Is user's consent on the BAP enough? How does the consent work in case of multiple lenders?

**Comment by Antriksh-Parmar** [comment]
@ankitagarwaladobe Lenders do not need any consent from the borrowers for soft pull. Technically it could work without borrowers providing an explicit consent, but we are still capturing it in the flow to be completely watertight.
